packages/react-charting/src/components/StackedBarChart/MultiStackedBarChart.base.tsx

```diff
@@ -335,7 +337,7 @@ export class MultiStackedBarChartBase extends React.Component<IMultiStackedBarCh
             textAnchor={this._isRTL ? 'end' : 'start'}
             y={barHeight / 2}
             dominantBaseline="central"
-            transform={`translate(${this._isRTL ? -4 : 4})`}
+            transform={`translate(${this._isRTL ? (barLabel < 100 ? -18 : -24) : 4})`}
```

Comment:
Is there any better way to get the amount of translation needed

Reply:
I figured out this issue its related to `textAnchor` which just have to keep it by default as `start`. instead of `end` does fixing the issue.